bitcoin/src/blockdata/witness.rs

```diff
@@ -166,26 +166,31 @@ impl Witness {
     }
 
     /// Creates [`Witness`] object from an array of byte-arrays
+    #[deprecated(since="0.30.0", note="use `witness::from_slice()` instead")]
```

Comment:
In 886b41904abd35a70aac23d27de29f2f92b6e854;

`Witness::from_slice` should be capitalized